### DISCUSSION TITLE
Stateful dataloader

### DISCUSSION THREAD

**Discussion by madey83** [idiap/coqui-ai-TTS#174] [Q&A] [answered]
Hello,

is there any way to be sure that all my dataset waves are used for training? How can i be sure if i stop training and start it again from 
(--restore_path) not used waves from dataset will be used from now on?

**Comment by eginhard** [comment, marked as answer]
This information is not stored, so it's not possible.

**Comment by madey83** [comment]
When I start fine-tuning process is it start using the same waves that they were used before or it continues from where it stopped?

I have 18hs of audio was and almost 100k steps done. I do not see any audio quality improvement. Sound is audible but choppy.

Model which I try to model is VITS based on pre-trained....

**Reply by eginhard** [reply]
> When I start fine-tuning process is it start using the same waves that they were used before or it continues from where it stopped?

As I wrote:
> This information is not stored, so it's not possible.

It also won't matter with that little data. Your issues likely come from somewhere else.

**Reply by madey83** [reply]
Ok. How to find place where issue could come from?

Is there any guide for troubleshooting?
